src/app/(views)/managed-courses/page.tsx

```diff
@@ -19,12 +19,23 @@ import toast from '@/utils/toast'
 import { Loading } from '@/components'
 import { cilPlus } from '@coreui/icons'
 import CIcon from '@coreui/icons-react'
+import ReactPaginate from 'react-paginate'
 
 export default function ManagedCourses() {
   const [deletingCourse, setDeletingCourse] = useState(false)
   const { courses, setCourses, fetchingCourses } = useGetCourses({ type: 'managed' })
   const [isDeleteModalVisible, setIsDeleteModalVisible] = useState(false)
 
+  const itemsPerPage = 8
+  const [currentPage, setCurrentPage] = useState(0)
+
+  const handlePageClick = (selectedPage: { selected: React.SetStateAction<number> }) => {
+    setCurrentPage(selectedPage.selected)
+  }
+
+  const offset = currentPage * itemsPerPage
+  const paginatedData = courses.slice(offset, offset + itemsPerPage)
```

Comment:
Just put all variables on top. This way they're together - that they don't feel lonely and we can see them at first glance.